client/controllers/dateSelector.coffee

```diff
@@ -0,0 +1,67 @@
+createInlineDateRangePicker = require '/imports/ui/inlineDateRangePicker'
+{ setVariables,
+  clearDateRange }          = require '/imports/ui/setRange'
```

Comment:
@nathanathan My bad, I changed the file name just before pushing and forgot to add the file. Sorry about that.